src/bindings/python/PyPlanarImageDesc.cpp

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+#include "PyImageDesc.h"
+
+namespace OCIO_NAMESPACE
+{
+
+void bindPyPlanarImageDesc(py::module & m)
+{
+    py::class_<PyPlanarImageDesc, PyImageDesc /* base */>(m, "PlanarImageDesc")
+        .def(py::init([](py::buffer & rData, 
+                         py::buffer & gData, 
+                         py::buffer & bData, 
+                         py::buffer & aData, 
```

Comment:
In the C++ API, the alpha pointer could be null.

Reply:
Instead of supporting `None` passed for alpha, I opted to add two more constructors that just don't take an alpha param. This felt like a more Pythonic option.